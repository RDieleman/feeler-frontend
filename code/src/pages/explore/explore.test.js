import React from "react";
import {shallow} from "enzyme";
import ExplorePage from "./explore.page";

it("Expect to render Explore page", () => {
    const moods = ["bad", "good", "worse"];

    expect(shallow(
        <ExplorePage
            moods={moods}
        />)).toMatchSnapshot();
});