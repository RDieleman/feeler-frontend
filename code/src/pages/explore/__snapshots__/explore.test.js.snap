// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Explore page 1`] = `ShallowWrapper {}`;
